### DISCUSSION TITLE
Stop Subsequent Validations After First Failure

### DISCUSSION THREAD

**Discussion by lu-yanpeng** [vueform/vueform#346] [Ideas]
Hi, I am a newcomer to `vueform`. When performing form validation, is there a way to stop the validation process immediately after the first rule fails?

```vue
<template>
    <text-element
      name="email"
      :rules="['email', registered]"
      />
</template>
```

In the above code, during email validation, even if it fails, the validation continues to the registered check. I think it would be better if subsequent validations do not proceed once the first validation fails. Is this a built-in feature of vueform, or can this behavior be customized?

**Comment by adamberecz** [comment]
Currently this isn't possible however if you have a custom rule you can access `this.element$?.invalid` in `check()` to determine if any of the rules failed before it.
